bootstrap/src/routes/Welcome.js

```diff
@@ -161,6 +161,32 @@ const Welcome = () => {
                 .
             </p>
 
+            <h3>Add multi-tenant functionality and optional project picker</h3>
+            <ul>
+                <li>
+                    In <Code>Home.js</Code> replace all hard-coded project ID's with the one provided by
```

Comment:
In <Code>Home.js</Code>, replace all hard-coded project IDs with the one provided by